app/components/publications/publication/proofs/proof-upload-modal.js

```diff
@@ -63,6 +64,7 @@ export default class PublicationsPublicationProofsProofUploadModalComponent exte
       file: this.file,
       name: this.name,
       receivedAtDate: this.receivedAtDate,
+      isProofIn : this.isProofIn,
```

Comment:
Merk op: deze component wordt 2x gebruikt. 1x voor het uploaden van de ontvangen drukproef. De code daarvoor is hieronder aangepast om naargelang de waarde van `isProofIn` de status te updaten. Echter ook door op `Drukproef > Documenten > Document uploaden > Brondocument uploaden` te klikken komt de modal met het vinkje "status updaten ... ?" tevoorschijn. Hier wordt bij bevestigen niets met de status gedaan. Het is waarschijnlijk niet wenselijk de status wél te veranderen, maar het vinkje kan verwarring zaaien. (met design?) te bekijken wat hier mee moet gebeuren.

Reply:
Ik ga ervan uit dat die upload-modal voor drukproeven verder aangepakt zal worden in 'KAS-3101 Pub. Detail - Drukproef - Drukproeven', want daar staan nog velden op die momenteel niet in lijn zijn met het design. En dat de modal om brondocumenten op te laden herwerkt wordt in 'KAS-3105  Pub. Detail - Besluit(en) - Referentie Documenten'. Ik zou deze PR daar niet op laten blokkeren.